Shared/Supporting Files/Extensions/Array+RawRepresentable.swift

```diff
@@ -15,10 +15,10 @@
 import Foundation
 
 /// An extension allowing an array to be used with the app storage property wrapper.
-extension Array: RawRepresentable where Element == String {
+extension Array where Element == String {
     /// Creates a new array from a given raw value.
     /// - Parameter rawValue: The raw value of the array to create.
-    public init(rawValue: String) {
+    init(rawValue: String) {
```

Comment:
This parameter should be renamed to `jsonString` as well. Also, please update doc.